content/en/serverless/glossary/_index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Serverless Glossary
+kind: documentation
+---
+
+This glossary focuses on terms and concepts specific to serverless architectures, cloud platform providers, and Datadog Serverless monitoring. 
+
+### General serverless concepts
+
+| Concept                         | Description                                                                                                                                                                                                          |
+|---------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Serverless          | A development model in which a cloud provider allocates backend resources on-demand. This paradigm enables developers to build, run, and deploy applications and services without having to manage infrastructure.                              |
+| Function          | In a serverless paradigm, a function is a self-contained application that runs in the cloud.                             |
+| Cloud-based development          | A workflow where developers run their code in the cloud during the development process, instead of just on their local machines. Generally, building serverless applications requires cloud-based development.                                |
+| Cold start          | The first time you invoke a particular function, it may take a longer time to run than in subsequent invocations. This phenomenon is known as a cold start, and may be caused by different factors: for instance, the cloud provider may need some time to provision underlying resources.                                |
+| Event-driven architecture          | An architectural pattern in which events drive communication between decoupled services.                                 |
+| Function-as-a-Service (FaaS)          | A subset of serverless functions. FaaS refers explicitly to event-driven paradigms.                               |
+
+## Cloud-specific concepts
```

Comment:
Not too sure we want to redefine the concepts already defined by the cloud providers. Perhaps it make more sense to link the user to the cloud provider's doc?

Reply:
I think it's helpful to consolidate here for a more full picture of what everything is — so people won't have to switch back and forth between our docs and AWS docs more than they need to